### PR TITLE
#1 init project

### PR DESCRIPTION
**Pull request by BadBrahmin**
- Issues #1 and #2 included in the PR.

### CODE REVIEW

File: feature/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+node_modules
```
Comment (harshaanNihal) [inline feature/.gitignore]: Try to keep an empty blank line after every file

File: feature/app.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
+const createError = require('http-errors');
+const express = require('express');
+const path = require('path');
+const cookieParser = require('cookie-parser');
+const logger = require('morgan');
+const mongoose = require('mongoose');
+
+const indexRouter = require('./routes/index');
+// const usersRouter = require('./routes/users');
+// const labelRouter = require('./routes/label');
```
Comment (harshaanNihal) [inline feature/app.js]: PR should only consist of code related to targeted issue.

File: feature/models/label.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  color: {
+    type: String,
+    required: true,
+  },
+  issues: [],
+});
+
+const Label = mongoose.model('Label', labelSchema);
+
+module.exports = Label;
```
Comment (harshaanNihal) [inline feature/models/label.js]: If a PR is Targetting More than 1 Issue than it should be mentioned in the PR description

Comment (BadBrahmin) [reply feature/models/label.js]: Description added.

File: feature/.gitignore (hunk 1 above)
Comment (BadBrahmin) [reply feature/.gitignore]: Only one entry on .gitignore right now but will keep that in mind.

File: feature/app.js (hunk 2 above)
Comment (BadBrahmin) [reply feature/app.js]: File updated.

**Pull request merged**